index.js

```diff
@@ -46,6 +50,14 @@ document.getElementById('submit_button').addEventListener('click', function () {
     user_feedback_elem.style.display = 'block'
 })
 
+document.getElementById('next_question_button').addEventListener('click', function () {
+    document.getElementById('reveal_button').disabled = false
+    document.getElementById('submit_button').disabled = false
+    get_new_question()
+    document.getElementById('next_question_button').disabled = true
+    document.getElementById('submit_button').disabled = false
```

Comment:
Since you're selecting the submit button twice here I'd suggest assigning it to a variable to make the code more DRY

Reply:
You could even take this a step further by putting any DOM queries (getElementByIds) that are repeated multiple times into variables in the global scope and then referencing those variables instead of reselecting them all the time. Again just to make things a bit more DRY - this one is optional though!